src/web/wfs/src/main/resources/GeoServerApplication.properties

```diff
@@ -50,7 +50,9 @@ WFSLayerConfig.otherSRS.message=A comma separated list of EPSG codes, e.g. 4326,
    override to a list of values specified in the WFS service configuration.
 
 WFSLayerConfig.coordinatesEncodingTitle=Coordinates Encoding
-WFSLayerConfig.coordinatesEncodingHelp.message=lalala
+WFSLayerConfig.coordinatesEncodingHelp.message= This settings control how geometries coordinates will be encoded in \
```

Comment:
I think it should be `This setting controls how geometry coordinates will be encoded in`